lib/src/ast_extensions.dart

```diff
@@ -88,40 +88,6 @@ extension AstIterableExtensions on Iterable<AstNode> {
 }
 
 extension ExpressionExtensions on Expression {
```

Comment:
Since this was only used in one place, I went ahead and inlined it there so that it was clearer what it applied to.